File: LDC.Atlas.WebApp/Client/app/shared/services/execution/dtos/month-end-temporary-adjustment-list-command.ts
import { MonthEndTemporaryAdjustmentCommand } from './monthend-temporary-adjustment-command';

export class MonthEndTemporaryAdjustmentListCommand {
    DocumentDate: Date;
    AccountingPeriod: Date;
    dataVersionId: number;
    dataVersionDate: Date;
    reportType: number;
}

File: LDC.Atlas.Services.Execution/Application/Commands/MonthEndTemporaryAdjustmentListCommand.cs
using LDC.Atlas.Services.Execution.Entities;
using MediatR;
using System;

namespace LDC.Atlas.Services.Execution.Application.Commands
{
    public class MonthEndTemporaryAdjustmentListCommand : IRequest<MonthEndTAResponse>
    {
        public DateTime DocumentDate { get; set; }

        public DateTime? AccountingPeriod { get; set; }

        internal string Company { get; set; }

        public int? DataVersionId { get; set; }

        public DateTime DataVersionDate { get; set; }

        public int? ReportType { get; set; }
    }
}
